app/js/reducers/createLabOrderReducer.js

```diff
@@ -0,0 +1,40 @@
+const initialState = {
+  labOrderData: {},
+  errorMessage: '',
+  status: {
+    error: false,
+    added: false,
+  },
+};
+
+const createLabOrderReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case 'CREATE_LAB_ORDER_SUCCESS': {
+      return {
+        ...state,
+        labOrderData: action.data,
+        status: {
+          added: true,
+          error: false,
+        },
+      };
+    }
+
+    case 'CREATE_LAB_ORDER_FAILURE': {
+      return {
+        ...state,
+        errorMessage: action.error,
+        status: {
+          error: true,
+          added: false,
+        },
+      };
+    }
```

Comment:
I see you are specifying the action types as strings in this reducer.
Though this would work fine, there is an `actionTypes.js` file that I believe should hold all the action types as constants for easy reusability.
I think adding the actionTypes to that file would be a nice refactoring.

Reply:
I get your point, if you look into the action creator for this PR, I imported the action types from the action types file, It is just my personal preference to not do that when I am working on reducers and on tests, everything in action remains in action.

Reply:
Awesome 👍 
Thanks for the clarification.